cli/ballerina-cli/src/main/resources/cli-help/ballerina-test.help

```diff
@@ -78,6 +78,14 @@ OPTIONS
            is an experimental feature which supports only a limited set of
            functionality.
 
+       --excludes
+            Specify the Ballerina source files/folders to be excluded from code coverage
+            calculation. Multiple source files/folders can be provided as a comma separated list.
+            Source file path can be absolute or relative to the package directory. '!' is used to omit
```

Comment:
Do we have a reference that we can point to?

Reply:
There is no such reference(it is a modified version of .gitignore). we may need to create one.